pkgs/os-specific/linux/kernel/common-config.nix

```diff
@@ -224,6 +224,17 @@ let
       UCLAMP_TASK_GROUP = yes;
     };
 
+    # All of these need to be enabled to support sched-ext
+    sched-ext = {
+      BPF = whenAtLeast "6.12" yes;
+      BPF_SYSCALL = yes;
+      SCHED_CLASS_EXT = whenAtLeast "6.12" yes;
+      BPF_JIT            = whenPlatformHasEBPFJit yes;
+      BPF_JIT_ALWAYS_ON  = whenPlatformHasEBPFJit (whenAtLeast "6.12" yes); # whenPlatformHasEBPFJit yes; # see https://github.com/NixOS/nixpkgs/issues/79304
+      BPF_JIT_DEFAULT_ON = whenPlatformHasEBPFJit yes;
```

Comment:
This is already set.

Reply:
I moved the option around, so it better represents which ones are required by sched-ext.

Reply:
It's not defined explicitly, it's set by default by our config scripts.